Libraries/CMakeLists.txt

```diff
@@ -24,6 +24,7 @@ add_subdirectory(pbxproj)
 add_subdirectory(pbxsetting)
```

Comment:
Each process-related commit seems better than the next.  Yay code cleanup.  LGTM